apps/nextjs-website/src/lib/strapi/fetches/fetchProducts.ts

```diff
@@ -2,17 +2,23 @@ import { fetchFromStrapi } from '@/lib/strapi/fetchFromStrapi';
 import { ProductsCodec } from '@/lib/strapi/codecs/ProductCodec';
 import qs from 'qs';
 
+export const productPopulate = {
+  populate: [
+    'logo',
+    'bannerLinks.icon',
+    'overview',
+    'quickstart_guide',
+    'api_data_list_page',
+    'api_data_list_page.apiData.*',
+    'api_data_list_page.apiData.apiRestDetail.*',
+    'guide_list_page',
+    'tutorial_list_page',
```

Comment:
We could export a const named `productRelationsPopulate` and use it here and in `fetchHomepage.ts`

Reply:
For now that query run only at build time and are cached i think is not necessarymaybe is better to add a comment to update in future if we pass to Standalone version